addon/templates/project/form.hbs

```diff
@@ -168,46 +168,86 @@
               @required={{true}}
             />
           </div>
-          {{#if this.project.buildingPermitIssueDate}}
+          {{#if 
+            (or 
+              this.project.buildingPermitIssueDate 
+              changeset.buildingPermitIssueDate 
+              this.import
+            )
```

Comment:
Hmm i keep thinking there has to be a nicer solution to prevent this template duplication for every field.